Airports/EGOV/Fixes.txt

```diff
@@ -0,0 +1,31 @@
+ANTUZ N053.24.40.670 W004.18.47.630
+ATWOX N053.17.56.970 W004.50.59.930
+AVLEZ N052.58.18.270 W004.35.25.250
+DEVHO N053.05.06.070 W004.42.31.010
+DODAV N053.26.52.540 W004.50.25.190
+EMZUN N053.03.48.680 W004.46.43.090
```

Comment:
```suggestion
;DODAV N053.26.52.540 W004.50.25.190 (elsewhere defined for EGOQ)
;EMZUN N053.03.48.680 W004.46.43.090 (elsewhere defined for EGOQ)
```
These two are defined already in [Airports](https://github.com/VATSIM-UK/UK-Sector-File/tree/fc664f7b165c57a6e0768cd733bd75dd2c72157b/Airports)/[EGOQ](https://github.com/VATSIM-UK/UK-Sector-File/tree/fc664f7b165c57a6e0768cd733bd75dd2c72157b/Airports/EGOQ)/Fixes.txt (hence why the compiler is failing)

Reply:
Yeah I've removed those two, should I add it as a comment?